rebuild.sh

```diff
@@ -87,6 +87,25 @@ umask $umask
 export MVN_UMASK=${umask}
 fetchSource
 
+DEFAULT_engine="docker"
+DEFAULT_engineopts_docker=""
+DEFAULT_engineopts_podman="--userns=keep-id"
+DEFAULT_engineopts="$([[ 'docker' == ${REBUILD_ENGINE:-$DEFAULT_engine} ]] && echo $DEFAULT_engineopts_docker || echo $DEFAULT_engineopts_podman)"
+DEFAULT_volumeflags=""
+
+if [ -z "${REBUILD_ENGINE}" ]
```

Comment:
there is already a `RB_SHELL` env variable (I can't say well documented...)

can we rename the `REBUILD_*` variables to something like `RB_OCI_*`?